entry/src/main/js/default/pages/index/index.js

```diff
@@ -127,4 +139,53 @@ export default {
     onWeekClick: function() {
         console.log("Week Button clicked !");
     },
+    fetchData : function(){
+        let data;
+        fetch.fetch({
+            url:'https://neumorphism-api.herokuapp.com/neumorphism/living_room',
```

Comment:
Try to get weather data from openweathermap.org and display it

Can refer to,
https://forums.developer.huawei.com/forumPortal/en/topic/0201541977865920174